lib/middleware/karma.js

```diff
@@ -175,6 +175,11 @@ var createKarmaMiddleware = function (
           var scriptTags = []
           var scriptUrls = []
           for (var i in files.included) {
+            if (!files.included.hasOwnProperty(i)) {
+                // files.included is not an array
+                // keys might not be an index
+              break
```

Comment:
Is it good idea to break loop?
In case if that strange file without path appears not at the end of "files.included" array, it will broke everything...
Maybe better to change 'break' to 'continue'?